packages/lesswrong/components/seasonal/solsticeSeason/solsiceSeasonConstants.ts

```diff
@@ -0,0 +1,14 @@
+export const CONTRAST_AMOUNT = 1;
+export const BRIGHTNESS_BOOST = 1;
+export const BRIGHTNESS_ADD = 0.05;
+// Rotation speed in radians per second (converted from 0.001 per frame at 60fps = 0.06 rad/s)
+export const ROTATION_SPEED = 0.06;
+
+export const DEFAULT_DAY_IMAGE_URL = "https://res.cloudinary.com/lesswrong-2-0/image/upload/v1761983935/flat_earth_Largest_still3_yltj4n.jpg";
+export const DEFAULT_NIGHT_IMAGE_URL = "https://res.cloudinary.com/lesswrong-2-0/image/upload/v1761984035/earth-night-light_kwpk53.jpg";
+export const DEFAULT_LUMINOSITY_IMAGE_URL = "https://res.cloudinary.com/lesswrong-2-0/image/upload/v1761947544/earth-night_fratqn.jpg";
```

Comment:
don't forget to check that these are reasonably-sized image files

Reply:
I did